### PR TITLE
Add routing to math magicians app

### PR DESCRIPTION
**Pull request by Atifzada**
### The following changes have been made to this PR:

- Added Home, Calculator, and Quotes Pages
- All pages are connected via routing react.
- UI improved.
- All Components are rendered on the right page.
- All linters error are removed

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -6,12 +6,13 @@ function Calculator(props) {
...
-        {calcState.result}
-        {calcState.function}
-        {calcState.next}
-      </div>
       <div className="btnsContainer">
+        <div className="display">
+          {calcState.result}
+          {calcState.function}
+          {calcState.next}
+        </div>
```
Comment (Munsa1) [inline src/components/Calculator.js]: - [ ] Great job so far, now even though were not supposed to fpay much attention to styling, we should follow the given layout of components in the given [wireframes](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md)

Given Wireframe
image

Your design
image

### _Kindly ensure your calculator is aligned to right and the text `Let's do some math` is present and aligned to the right in alignment with the top of the calculator_

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Atifzada, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Used Routing :heavy_check_mark: 
Descriptive Pr :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Atifzada) [reply src/components/Calculator.js]: Thank you for the helpful review. Surely I will come up with solution

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Atif, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job following the instructions of the previous CR.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**